hash/gbcolor.xml

```diff
@@ -27208,6 +27208,18 @@ license:CC0
 			</dataarea>
 		</part>
 	</software>
+	
+	<software name="nuwang2k" supported="no">
+		<description>Nü Wang Ge Dou 2000</description>
```

Comment:
You should do Hanyu Pinyin properly (Nǚwáng Gédòu), and put the Chinese title in an `alt_title`.

Reply:
TBF, I just copied the title from the page where the dump was found.

Reply:
Copy/pasting wrong things doesn’t make them less wrong.